pkg/influx/api.go

```diff
@@ -19,8 +19,11 @@ type API struct {
 
 func (a *API) Register(router *mux.Router) {
 	registerer := route.NewMuxRegisterer(router)
-
-	registerer.RegisterRoute("/api/v1/push/influx/write", http.HandlerFunc(a.handleSeriesPush), http.MethodPost)
+	registerer.RegisterRoute("/api/v2/write", http.HandlerFunc(a.handleSeriesPush), http.MethodPost)
+	registerer.RegisterRoute("/healthz", http.HandlerFunc(func(rw http.ResponseWriter, req *http.Request) {
```

Comment:
I think Justin added healthz in a different PR (https://github.com/grafana/influx2cortex/pull/51), let's make sure this isn't redundant.

Reply:
Good catch, thanks